/jds/bin/diff -uprN gnome-panel-2.26.3.old/icons/Makefile.am gnome-panel-2.26.3/icons/Makefile.am
--- gnome-panel-2.26.3.old/icons/Makefile.am	2009-07-01 17:51:31.339937000 +0100
+++ gnome-panel-2.26.3/icons/Makefile.am	2009-07-01 18:13:42.339203000 +0100
@@ -7,6 +7,7 @@ gegl_DATA = 				\
 	gnome-gegl2-2.png		\
 	gnome-gegl2.png			\
 	world_map-960.png		\
+	bottom-panel-image.png		\
 	$(NULL)
 
 EXTRA_DIST = $(gegl_DATA)
/jds/bin/diff -uprN gnome-panel-2.26.3.old/gnome-panel/panel-default-setup.entries gnome-panel-2.26.3/gnome-panel/panel-default-setup.entries
--- gnome-panel-2.26.3.old/gnome-panel/panel-default-setup.entries	2009-07-01 17:51:28.824197000 +0100
+++ gnome-panel-2.26.3/gnome-panel/panel-default-setup.entries	2009-07-01 18:13:11.140607000 +0100
@@ -10,9 +10,6 @@
       <value>
         <list type="string">
           <value>
-            <string>top_panel</string>
-          </value>
-          <value>
             <string>bottom_panel</string>
           </value>
         </list>
@@ -27,13 +24,7 @@
       <value>
         <list type="string">
           <value>
-            <string>menu_bar</string>
-          </value>
-          <value>
-            <string>browser_launcher</string>
-          </value>
-          <value>
-            <string>email_launcher</string>
+            <string>main_menu</string>
           </value>
         </list>
       </value>
@@ -47,76 +38,30 @@
       <value>
         <list type="string">
           <value>
-            <string>window_menu</string>
+            <string>clock</string>
           </value>
           <value>
-            <string>clock</string>
+            <string>show_desktop</string>
           </value>
           <value>
-            <string>notification_area</string>
+            <string>window_list</string>
           </value>
           <value>
-            <string>show_desktop_button</string>
+            <string>workspace_switcher</string>
           </value>
           <value>
-            <string>window_list</string>
+            <string>notification_area</string>
           </value>
           <value>
-            <string>workspace_switcher</string>
+            <string>mixer</string>
+          </value>
+          <value>
+            <string>trash</string>
           </value>
         </list>
       </value>
     </entry>
 
-  <!-- Top Panel -->
-
-    <entry>
-      <key>toplevels/top_panel/expand</key>
-      <schema_key>/schemas/apps/panel/toplevels/expand</schema_key>
-      <value>
-        <bool>true</bool>
-      </value>
-    </entry>
-    <entry>
-      <key>toplevels/top_panel/orientation</key>
-      <schema_key>/schemas/apps/panel/toplevels/orientation</schema_key>
-      <value>
-        <string>top</string>
-      </value>
-    </entry>
-    <entry>
-      <key>toplevels/top_panel/size</key>
-      <schema_key>/schemas/apps/panel/toplevels/size</schema_key>
-      <value>
-        <int>24</int>
-      </value>
-    </entry>
-
-    <entry><key>toplevels/top_panel/name</key><schema_key>/schemas/apps/panel/toplevels/name</schema_key></entry>
-    <entry><key>toplevels/top_panel/screen</key><schema_key>/schemas/apps/panel/toplevels/screen</schema_key></entry>
-    <entry><key>toplevels/top_panel/monitor</key><schema_key>/schemas/apps/panel/toplevels/monitor</schema_key></entry>
-    <entry><key>toplevels/top_panel/x</key><schema_key>/schemas/apps/panel/toplevels/x</schema_key></entry>
-    <entry><key>toplevels/top_panel/y</key><schema_key>/schemas/apps/panel/toplevels/y</schema_key></entry>
-    <entry><key>toplevels/top_panel/x_right</key><schema_key>/schemas/apps/panel/toplevels/x_right</schema_key></entry>
-    <entry><key>toplevels/top_panel/y_bottom</key><schema_key>/schemas/apps/panel/toplevels/y_bottom</schema_key></entry>
-    <entry><key>toplevels/top_panel/x_centered</key><schema_key>/schemas/apps/panel/toplevels/x_centered</schema_key></entry>
-    <entry><key>toplevels/top_panel/y_centered</key><schema_key>/schemas/apps/panel/toplevels/y_centered</schema_key></entry>
-    <entry><key>toplevels/top_panel/auto_hide</key><schema_key>/schemas/apps/panel/toplevels/auto_hide</schema_key></entry>
-    <entry><key>toplevels/top_panel/enable_animations</key><schema_key>/schemas/apps/panel/toplevels/enable_animations</schema_key></entry>
-    <entry><key>toplevels/top_panel/enable_buttons</key><schema_key>/schemas/apps/panel/toplevels/enable_buttons</schema_key></entry>
-    <entry><key>toplevels/top_panel/enable_arrows</key><schema_key>/schemas/apps/panel/toplevels/enable_arrows</schema_key></entry>
-    <entry><key>toplevels/top_panel/hide_delay</key><schema_key>/schemas/apps/panel/toplevels/hide_delay</schema_key></entry>
-    <entry><key>toplevels/top_panel/unhide_delay</key><schema_key>/schemas/apps/panel/toplevels/unhide_delay</schema_key></entry>
-    <entry><key>toplevels/top_panel/auto_hide_size</key><schema_key>/schemas/apps/panel/toplevels/auto_hide_size</schema_key></entry>
-    <entry><key>toplevels/top_panel/animation_speed</key><schema_key>/schemas/apps/panel/toplevels/animation_speed</schema_key></entry>
-    <entry><key>toplevels/top_panel/background/type</key><schema_key>/schemas/apps/panel/toplevels/background/type</schema_key></entry>
-    <entry><key>toplevels/top_panel/background/color</key><schema_key>/schemas/apps/panel/toplevels/background/color</schema_key></entry>
-    <entry><key>toplevels/top_panel/background/opacity</key><schema_key>/schemas/apps/panel/toplevels/background/opacity</schema_key></entry>
-    <entry><key>toplevels/top_panel/background/image</key><schema_key>/schemas/apps/panel/toplevels/background/image</schema_key></entry>
-    <entry><key>toplevels/top_panel/background/fit</key><schema_key>/schemas/apps/panel/toplevels/background/fit</schema_key></entry>
-    <entry><key>toplevels/top_panel/background/stretch</key><schema_key>/schemas/apps/panel/toplevels/background/stretch</schema_key></entry>
-    <entry><key>toplevels/top_panel/background/rotate</key><schema_key>/schemas/apps/panel/toplevels/background/rotate</schema_key></entry>
-
   <!-- Bottom Panel -->
 
     <entry>
@@ -137,7 +82,7 @@
       <key>toplevels/bottom_panel/size</key>
       <schema_key>/schemas/apps/panel/toplevels/size</schema_key>
       <value>
-        <int>24</int>
+        <int>32</int>
       </value>
     </entry>
     <entry>
@@ -164,277 +109,314 @@
     <entry><key>toplevels/bottom_panel/unhide_delay</key><schema_key>/schemas/apps/panel/toplevels/unhide_delay</schema_key></entry>
     <entry><key>toplevels/bottom_panel/auto_hide_size</key><schema_key>/schemas/apps/panel/toplevels/auto_hide_size</schema_key></entry>
     <entry><key>toplevels/bottom_panel/animation_speed</key><schema_key>/schemas/apps/panel/toplevels/animation_speed</schema_key></entry>
-    <entry><key>toplevels/bottom_panel/background/type</key><schema_key>/schemas/apps/panel/toplevels/background/type</schema_key></entry>
+
+
+     <entry>
+      <key>toplevels/bottom_panel/background/type</key>
+      <schema_key>/schemas/apps/panel/toplevels/background/type</schema_key>
+       <value>
+        <string>image</string>
+       </value>
+     </entry>
     <entry><key>toplevels/bottom_panel/background/color</key><schema_key>/schemas/apps/panel/toplevels/background/color</schema_key></entry>
     <entry><key>toplevels/bottom_panel/background/opacity</key><schema_key>/schemas/apps/panel/toplevels/background/opacity</schema_key></entry>
-    <entry><key>toplevels/bottom_panel/background/image</key><schema_key>/schemas/apps/panel/toplevels/background/image</schema_key></entry>
+     <entry>
+      <key>toplevels/bottom_panel/background/image</key>
+      <schema_key>/schemas/apps/panel/toplevels/background/image</schema_key>
+       <value>
+        <string>/usr/share/gnome-panel/pixmaps/bottom-panel-image.png</string>
+       </value>
+     </entry>
     <entry><key>toplevels/bottom_panel/background/fit</key><schema_key>/schemas/apps/panel/toplevels/background/fit</schema_key></entry>
-    <entry><key>toplevels/bottom_panel/background/stretch</key><schema_key>/schemas/apps/panel/toplevels/background/stretch</schema_key></entry>
-    <entry><key>toplevels/bottom_panel/background/rotate</key><schema_key>/schemas/apps/panel/toplevels/background/rotate</schema_key></entry>
-
-  <!-- Menu Bar -->
-
-    <entry>
-      <key>objects/menu_bar/object_type</key>
-      <schema_key>/schemas/apps/panel/objects/object_type</schema_key>
-      <value>
-        <string>menu-bar</string>
-      </value>
-    </entry>
-    <entry>
-      <key>objects/menu_bar/toplevel_id</key>
-      <schema_key>/schemas/apps/panel/objects/toplevel_id</schema_key>
-      <value>
-        <string>top_panel</string>
-      </value>
-    </entry>
-    <entry>
-      <key>objects/menu_bar/position</key>
-      <schema_key>/schemas/apps/panel/objects/position</schema_key>
-      <value>
-        <int>0</int>
-      </value>
-    </entry>
-    <entry>
-      <key>objects/menu_bar/panel_right_stick</key>
-      <schema_key>/schemas/apps/panel/objects/panel_right_stick</schema_key>
-      <value>
-        <bool>false</bool>
-      </value>
-    </entry>
-    <entry>
-      <key>objects/menu_bar/locked</key>
-      <schema_key>/schemas/apps/panel/objects/locked</schema_key>
+    <entry><key>toplevels/bottom_panel/background/stretch</key><schema_key>/schemas/apps/panel/toplevels/background/stretch</schema_key>
+       <value>
+        <bool>true</bool>
+       </value>
+     </entry>
+    <entry><key>toplevels/bottom_panel/background/rotate</key><schema_key>/schemas/apps/panel/toplevels/background/rotate</schema_key>
       <value>
         <bool>true</bool>
       </value>
     </entry>
 
-    <entry><key>objects/menu_bar/bonobo_iid</key><schema_key>/schemas/apps/panel/objects/bonobo_iid</schema_key></entry>
-    <entry><key>objects/menu_bar/attached_toplevel_id</key><schema_key>/schemas/apps/panel/objects/attached_toplevel_id</schema_key> </entry>
-    <entry><key>objects/menu_bar/tooltip</key><schema_key>/schemas/apps/panel/objects/tooltip</schema_key></entry>
-    <entry><key>objects/menu_bar/use_custom_icon</key><schema_key>/schemas/apps/panel/objects/use_custom_icon</schema_key></entry>
-    <entry><key>objects/menu_bar/custom_icon</key><schema_key>/schemas/apps/panel/objects/custom_icon</schema_key></entry>
-    <entry><key>objects/menu_bar/use_menu_path</key><schema_key>/schemas/apps/panel/objects/use_menu_path</schema_key></entry>
-    <entry><key>objects/menu_bar/menu_path</key><schema_key>/schemas/apps/panel/objects/menu_path</schema_key></entry>
-    <entry><key>objects/menu_bar/launcher_location</key><schema_key>/schemas/apps/panel/objects/launcher_location</schema_key></entry>
-    <entry><key>objects/menu_bar/action_type</key><schema_key>/schemas/apps/panel/objects/action_type</schema_key></entry>
-
-  <!-- Browser Launcher -->
+  <!-- Main Menu -->
 
     <entry>
-      <key>objects/browser_launcher/object_type</key>
+      <key>objects/main_menu/object_type</key>
       <schema_key>/schemas/apps/panel/objects/object_type</schema_key>
       <value>
-        <string>launcher-object</string>
+        <string>menu-object</string>
       </value>
     </entry>
     <entry>
-      <key>objects/browser_launcher/toplevel_id</key>
+      <key>objects/main_menu/toplevel_id</key>
       <schema_key>/schemas/apps/panel/objects/toplevel_id</schema_key>
       <value>
-        <string>top_panel</string>
+        <string>bottom_panel</string>
       </value>
     </entry>
     <entry>
-      <key>objects/browser_launcher/position</key>
+      <key>objects/main_menu/position</key>
       <schema_key>/schemas/apps/panel/objects/position</schema_key>
       <value>
-        <int>1</int>
+        <int>0</int>
       </value>
     </entry>
     <entry>
-      <key>objects/browser_launcher/panel_right_stick</key>
+      <key>objects/main_menu/panel_right_stick</key>
       <schema_key>/schemas/apps/panel/objects/panel_right_stick</schema_key>
       <value>
         <bool>false</bool>
       </value>
     </entry>
     <entry>
-      <key>objects/browser_launcher/locked</key>
+      <key>objects/main_menu/locked</key>
       <schema_key>/schemas/apps/panel/objects/locked</schema_key>
       <value>
         <bool>true</bool>
       </value>
     </entry>
-    <entry>
-      <key>objects/browser_launcher/launcher_location</key>
-      <schema_key>/schemas/apps/panel/objects/launcher_location</schema_key>
-      <value>
-        <string>epiphany.desktop</string>
-      </value>
-    </entry>
 
-    <entry><key>objects/browser_launcher/bonobo_iid</key><schema_key>/schemas/apps/panel/objects/bonobo_iid</schema_key></entry>
-    <entry><key>objects/browser_launcher/attached_toplevel_id</key><schema_key>/schemas/apps/panel/objects/attached_toplevel_id</schema_key> </entry>
-    <entry><key>objects/browser_launcher/tooltip</key><schema_key>/schemas/apps/panel/objects/tooltip</schema_key></entry>
-    <entry><key>objects/browser_launcher/use_custom_icon</key><schema_key>/schemas/apps/panel/objects/use_custom_icon</schema_key></entry>
-    <entry><key>objects/browser_launcher/custom_icon</key><schema_key>/schemas/apps/panel/objects/custom_icon</schema_key></entry>
-    <entry><key>objects/browser_launcher/use_menu_path</key><schema_key>/schemas/apps/panel/objects/use_menu_path</schema_key></entry>
-    <entry><key>objects/browser_launcher/menu_path</key><schema_key>/schemas/apps/panel/objects/menu_path</schema_key></entry>
-    <entry><key>objects/browser_launcher/action_type</key><schema_key>/schemas/apps/panel/objects/action_type</schema_key></entry>
+    <entry><key>objects/main_menu/bonobo_iid</key><schema_key>/schemas/apps/panel/objects/bonobo_iid</schema_key></entry>
+    <entry><key>objects/main_menu/attached_toplevel_id</key><schema_key>/schemas/apps/panel/objects/attached_toplevel_id</schema_key> </entry>
+    <entry><key>objects/main_menu/tooltip</key><schema_key>/schemas/apps/panel/objects/tooltip</schema_key></entry>
+    <entry><key>objects/main_menu/use_custom_icon</key><schema_key>/schemas/apps/panel/objects/use_custom_icon</schema_key></entry>
+    <entry><key>objects/main_menu/custom_icon</key><schema_key>/schemas/apps/panel/objects/custom_icon</schema_key></entry>
+    <entry><key>objects/main_menu/use_menu_path</key><schema_key>/schemas/apps/panel/objects/use_menu_path</schema_key></entry>
+    <entry><key>objects/main_menu/menu_path</key><schema_key>/schemas/apps/panel/objects/menu_path</schema_key></entry>
+    <entry><key>objects/main_menu/launcher_location</key><schema_key>/schemas/apps/panel/objects/launcher_location</schema_key></entry>
+    <entry><key>objects/main_menu/action_type</key><schema_key>/schemas/apps/panel/objects/action_type</schema_key></entry>
+    <entry><key>objects/main_menu/text</key><schema_key>/schemas/apps/panel/objects/text</schema_key></entry>
+    <entry><key>objects/main_menu/text_font</key><schema_key>/schemas/apps/panel/objects/text_font</schema_key></entry>
+    <entry><key>objects/main_menu/boundary_box</key><schema_key>/schemas/apps/panel/objects/boundary_box</schema_key></entry>
 
-  <!-- Email Launcher -->
+  <!-- Clock Applet -->
 
     <entry>
-      <key>objects/email_launcher/object_type</key>
+      <key>objects/clock/object_type</key>
       <schema_key>/schemas/apps/panel/objects/object_type</schema_key>
       <value>
-        <string>launcher-object</string>
+        <string>bonobo-object</string>
       </value>
     </entry>
     <entry>
-      <key>objects/email_launcher/toplevel_id</key>
+      <key>objects/clock/toplevel_id</key>
       <schema_key>/schemas/apps/panel/objects/toplevel_id</schema_key>
       <value>
-        <string>top_panel</string>
+        <string>bottom_panel</string>
       </value>
     </entry>
     <entry>
-      <key>objects/email_launcher/position</key>
+      <key>objects/clock/position</key>
       <schema_key>/schemas/apps/panel/objects/position</schema_key>
       <value>
-        <int>2</int>
+        <int>1</int>
       </value>
     </entry>
     <entry>
-      <key>objects/email_launcher/panel_right_stick</key>
+      <key>objects/clock/panel_right_stick</key>
       <schema_key>/schemas/apps/panel/objects/panel_right_stick</schema_key>
       <value>
         <bool>false</bool>
       </value>
     </entry>
     <entry>
-      <key>objects/email_launcher/locked</key>
+      <key>objects/clock/locked</key>
       <schema_key>/schemas/apps/panel/objects/locked</schema_key>
       <value>
         <bool>true</bool>
       </value>
     </entry>
     <entry>
-      <key>objects/email_launcher/launcher_location</key>
-      <schema_key>/schemas/apps/panel/objects/launcher_location</schema_key>
+      <key>objects/clock/bonobo_iid</key>
+      <schema_key>/schemas/apps/panel/objects/bonobo_iid</schema_key>
       <value>
-        <string>evolution.desktop</string>
+        <string>OAFID:GNOME_ClockApplet</string>
       </value>
     </entry>
 
-    <entry><key>objects/email_launcher/bonobo_iid</key><schema_key>/schemas/apps/panel/objects/bonobo_iid</schema_key></entry>
-    <entry><key>objects/email_launcher/attached_toplevel_id</key><schema_key>/schemas/apps/panel/objects/attached_toplevel_id</schema_key> </entry>
-    <entry><key>objects/email_launcher/tooltip</key><schema_key>/schemas/apps/panel/objects/tooltip</schema_key></entry>
-    <entry><key>objects/email_launcher/use_custom_icon</key><schema_key>/schemas/apps/panel/objects/use_custom_icon</schema_key></entry>
-    <entry><key>objects/email_launcher/custom_icon</key><schema_key>/schemas/apps/panel/objects/custom_icon</schema_key></entry>
-    <entry><key>objects/email_launcher/use_menu_path</key><schema_key>/schemas/apps/panel/objects/use_menu_path</schema_key></entry>
-    <entry><key>objects/email_launcher/menu_path</key><schema_key>/schemas/apps/panel/objects/menu_path</schema_key></entry>
-    <entry><key>objects/email_launcher/action_type</key><schema_key>/schemas/apps/panel/objects/action_type</schema_key></entry>
+    <entry><key>applets/clock/attached_toplevel_id</key><schema_key>/schemas/apps/panel/objects/attached_toplevel_id</schema_key> </entry>
+    <entry><key>applets/clock/tooltip</key><schema_key>/schemas/apps/panel/objects/tooltip</schema_key></entry>
+    <entry><key>applets/clock/use_custom_icon</key><schema_key>/schemas/apps/panel/objects/use_custom_icon</schema_key></entry>
+    <entry><key>applets/clock/custom_icon</key><schema_key>/schemas/apps/panel/objects/custom_icon</schema_key></entry>
+    <entry><key>applets/clock/use_menu_path</key><schema_key>/schemas/apps/panel/objects/use_menu_path</schema_key></entry>
+    <entry><key>applets/clock/menu_path</key><schema_key>/schemas/apps/panel/objects/menu_path</schema_key></entry>
+    <entry><key>applets/clock/launcher_location</key><schema_key>/schemas/apps/panel/objects/launcher_location</schema_key></entry>
+    <entry><key>applets/clock/action_type</key><schema_key>/schemas/apps/panel/objects/action_type</schema_key></entry>
+    <entry><key>applets/clock/text</key><schema_key>/schemas/apps/panel/objects/text</schema_key></entry>
+    <entry><key>applets/clock/text_font</key><schema_key>/schemas/apps/panel/objects/text_font</schema_key></entry>
+    <entry><key>applets/clock/boundary_box</key><schema_key>/schemas/apps/panel/objects/boundary_box</schema_key></entry>
 
-  <!-- Window Menu Applet -->
+  <!-- Show Desktop Applet -->
 
     <entry>
-      <key>applets/window_menu/object_type</key>
+      <key>applets/show_desktop/object_type</key>
       <schema_key>/schemas/apps/panel/objects/object_type</schema_key>
       <value>
         <string>bonobo-applet</string>
       </value>
     </entry>
     <entry>
-      <key>applets/window_menu/toplevel_id</key>
+      <key>applets/show_desktop/toplevel_id</key>
       <schema_key>/schemas/apps/panel/objects/toplevel_id</schema_key>
       <value>
-        <string>top_panel</string>
+        <string>bottom_panel</string>
       </value>
     </entry>
     <entry>
-      <key>applets/window_menu/position</key>
+      <key>applets/show_desktop/position</key>
       <schema_key>/schemas/apps/panel/objects/position</schema_key>
       <value>
-        <int>0</int>
+        <int>2</int>
       </value>
     </entry>
     <entry>
-      <key>applets/window_menu/panel_right_stick</key>
+      <key>applets/show_desktop/panel_right_stick</key>
       <schema_key>/schemas/apps/panel/objects/panel_right_stick</schema_key>
       <value>
-        <bool>true</bool>
+        <bool>false</bool>
       </value>
     </entry>
     <entry>
-      <key>applets/window_menu/locked</key>
+      <key>applets/show_desktop/locked</key>
       <schema_key>/schemas/apps/panel/objects/locked</schema_key>
       <value>
         <bool>true</bool>
       </value>
     </entry>
     <entry>
-      <key>applets/window_menu/bonobo_iid</key>
+      <key>applets/show_desktop/bonobo_iid</key>
       <schema_key>/schemas/apps/panel/objects/bonobo_iid</schema_key>
       <value>
-        <string>OAFIID:GNOME_WindowMenuApplet</string>
+        <string>OAFIID:GNOME_ShowDesktopApplet</string>
       </value>
     </entry>
 
-    <entry><key>applets/window_menu/attached_toplevel_id</key><schema_key>/schemas/apps/panel/objects/attached_toplevel_id</schema_key> </entry>
-    <entry><key>applets/window_menu/tooltip</key><schema_key>/schemas/apps/panel/objects/tooltip</schema_key></entry>
-    <entry><key>applets/window_menu/use_custom_icon</key><schema_key>/schemas/apps/panel/objects/use_custom_icon</schema_key></entry>
-    <entry><key>applets/window_menu/custom_icon</key><schema_key>/schemas/apps/panel/objects/custom_icon</schema_key></entry>
-    <entry><key>applets/window_menu/use_menu_path</key><schema_key>/schemas/apps/panel/objects/use_menu_path</schema_key></entry>
-    <entry><key>applets/window_menu/menu_path</key><schema_key>/schemas/apps/panel/objects/menu_path</schema_key></entry>
-    <entry><key>applets/window_menu/launcher_location</key><schema_key>/schemas/apps/panel/objects/launcher_location</schema_key></entry>
-    <entry><key>applets/window_menu/action_type</key><schema_key>/schemas/apps/panel/objects/action_type</schema_key></entry>
+    <entry><key>applets/show_desktop/attached_toplevel_id</key><schema_key>/schemas/apps/panel/objects/attached_toplevel_id</schema_key> </entry>
+    <entry><key>applets/show_desktop/tooltip</key><schema_key>/schemas/apps/panel/objects/tooltip</schema_key></entry>
+    <entry><key>applets/show_desktop/use_custom_icon</key><schema_key>/schemas/apps/panel/objects/use_custom_icon</schema_key></entry>
+    <entry><key>applets/show_desktop/custom_icon</key><schema_key>/schemas/apps/panel/objects/custom_icon</schema_key></entry>
+    <entry><key>applets/show_desktop/use_menu_path</key><schema_key>/schemas/apps/panel/objects/use_menu_path</schema_key></entry>
+    <entry><key>applets/show_desktop/menu_path</key><schema_key>/schemas/apps/panel/objects/menu_path</schema_key></entry>
+    <entry><key>applets/show_desktop/launcher_location</key><schema_key>/schemas/apps/panel/objects/launcher_location</schema_key></entry>
+    <entry><key>applets/show_desktop/action_type</key><schema_key>/schemas/apps/panel/objects/action_type</schema_key></entry>
+    <entry><key>applets/show_desktop/text</key><schema_key>/schemas/apps/panel/objects/text</schema_key></entry>
+    <entry><key>applets/show_desktop/text_font</key><schema_key>/schemas/apps/panel/objects/text_font</schema_key></entry>
+    <entry><key>applets/show_desktop/boundary_box</key><schema_key>/schemas/apps/panel/objects/boundary_box</schema_key></entry>
 
-  <!-- Clock Applet -->
+  <!-- Window List Applet -->
+
+     <entry>
+      <key>applets/window_list/object_type</key>
+       <schema_key>/schemas/apps/panel/objects/object_type</schema_key>
+       <value>
+         <string>bonobo-applet</string>
+       </value>
+     </entry>
+     <entry>
+      <key>applets/window_list/toplevel_id</key>
+       <schema_key>/schemas/apps/panel/objects/toplevel_id</schema_key>
+       <value>
+        <string>bottom_panel</string>
+       </value>
+     </entry>
+     <entry>
+      <key>applets/window_list/position</key>
+       <schema_key>/schemas/apps/panel/objects/position</schema_key>
+       <value>
+        <int>3</int>
+       </value>
+     </entry>
+     <entry>
+      <key>applets/window_list/panel_right_stick</key>
+       <schema_key>/schemas/apps/panel/objects/panel_right_stick</schema_key>
+       <value>
+        <bool>false</bool>
+       </value>
+     </entry>
+     <entry>
+      <key>applets/window_list/locked</key>
+       <schema_key>/schemas/apps/panel/objects/locked</schema_key>
+       <value>
+         <bool>true</bool>
+       </value>
+     </entry>
+     <entry>
+      <key>applets/window_list/bonobo_iid</key>
+       <schema_key>/schemas/apps/panel/objects/bonobo_iid</schema_key>
+       <value>
+        <string>OAFIID:GNOME_WindowListApplet</string>
+       </value>
+     </entry>
+ 
+    <entry><key>applets/window_list/attached_toplevel_id</key><schema_key>/schemas/apps/panel/objects/attached_toplevel_id</schema_key> </entry>
+    <entry><key>applets/window_list/tooltip</key><schema_key>/schemas/apps/panel/objects/tooltip</schema_key></entry>
+    <entry><key>applets/window_list/use_custom_icon</key><schema_key>/schemas/apps/panel/objects/use_custom_icon</schema_key></entry>
+    <entry><key>applets/window_list/custom_icon</key><schema_key>/schemas/apps/panel/objects/custom_icon</schema_key></entry>
+    <entry><key>applets/window_list/use_menu_path</key><schema_key>/schemas/apps/panel/objects/use_menu_path</schema_key></entry>
+    <entry><key>applets/window_list/menu_path</key><schema_key>/schemas/apps/panel/objects/menu_path</schema_key></entry>
+    <entry><key>applets/window_list/launcher_location</key><schema_key>/schemas/apps/panel/objects/launcher_location</schema_key></entry>
+    <entry><key>applets/window_list/action_type</key><schema_key>/schemas/apps/panel/objects/action_type</schema_key></entry>
+    <entry><key>applets/window_list/text</key><schema_key>/schemas/apps/panel/objects/text</schema_key></entry>
+    <entry><key>applets/window_list/text_font</key><schema_key>/schemas/apps/panel/objects/text_font</schema_key></entry>
+    <entry><key>applets/window_list/boundary_box</key><schema_key>/schemas/apps/panel/objects/boundary_box</schema_key></entry>
+
+  <!-- Workspace Switcher Applet -->
 
     <entry>
-      <key>applets/clock/object_type</key>
+      <key>applets/workspace_switcher/object_type</key>
       <schema_key>/schemas/apps/panel/objects/object_type</schema_key>
       <value>
         <string>bonobo-applet</string>
       </value>
     </entry>
     <entry>
-      <key>applets/clock/toplevel_id</key>
+      <key>applets/workspace_switcher/toplevel_id</key>
       <schema_key>/schemas/apps/panel/objects/toplevel_id</schema_key>
       <value>
-        <string>top_panel</string>
+        <string>bottom_panel</string>
       </value>
     </entry>
     <entry>
-      <key>applets/clock/position</key>
+      <key>applets/workspace_switcher/position</key>
       <schema_key>/schemas/apps/panel/objects/position</schema_key>
       <value>
-        <int>2</int>
+        <int>3</int>
       </value>
     </entry>
     <entry>
-      <key>applets/clock/panel_right_stick</key>
+      <key>applets/workspace_switcher/panel_right_stick</key>
       <schema_key>/schemas/apps/panel/objects/panel_right_stick</schema_key>
       <value>
         <bool>true</bool>
       </value>
     </entry>
     <entry>
-      <key>applets/clock/locked</key>
+      <key>applets/workspace_switcher/locked</key>
       <schema_key>/schemas/apps/panel/objects/locked</schema_key>
       <value>
         <bool>true</bool>
       </value>
     </entry>
     <entry>
-      <key>applets/clock/bonobo_iid</key>
+      <key>applets/workspace_switcher/bonobo_iid</key>
       <schema_key>/schemas/apps/panel/objects/bonobo_iid</schema_key>
       <value>
-        <string>OAFIID:GNOME_ClockApplet</string>
+        <string>OAFIID:GNOME_WorkspaceSwitcherApplet</string>
       </value>
     </entry>
 
-    <entry><key>applets/clock/attached_toplevel_id</key><schema_key>/schemas/apps/panel/objects/attached_toplevel_id</schema_key> </entry>
-    <entry><key>applets/clock/tooltip</key><schema_key>/schemas/apps/panel/objects/tooltip</schema_key></entry>
-    <entry><key>applets/clock/use_custom_icon</key><schema_key>/schemas/apps/panel/objects/use_custom_icon</schema_key></entry>
-    <entry><key>applets/clock/custom_icon</key><schema_key>/schemas/apps/panel/objects/custom_icon</schema_key></entry>
-    <entry><key>applets/clock/use_menu_path</key><schema_key>/schemas/apps/panel/objects/use_menu_path</schema_key></entry>
-    <entry><key>applets/clock/menu_path</key><schema_key>/schemas/apps/panel/objects/menu_path</schema_key></entry>
-    <entry><key>applets/clock/launcher_location</key><schema_key>/schemas/apps/panel/objects/launcher_location</schema_key></entry>
-    <entry><key>applets/clock/action_type</key><schema_key>/schemas/apps/panel/objects/action_type</schema_key></entry>
+    <entry><key>applets/workspace_switcher/attached_toplevel_id</key><schema_key>/schemas/apps/panel/objects/attached_toplevel_id</schema_key> </entry>
+    <entry><key>applets/workspace_switcher/tooltip</key><schema_key>/schemas/apps/panel/objects/tooltip</schema_key></entry>
+    <entry><key>applets/workspace_switcher/use_custom_icon</key><schema_key>/schemas/apps/panel/objects/use_custom_icon</schema_key></entry>
+    <entry><key>applets/workspace_switcher/custom_icon</key><schema_key>/schemas/apps/panel/objects/custom_icon</schema_key></entry>
+    <entry><key>applets/workspace_switcher/use_menu_path</key><schema_key>/schemas/apps/panel/objects/use_menu_path</schema_key></entry>
+    <entry><key>applets/workspace_switcher/menu_path</key><schema_key>/schemas/apps/panel/objects/menu_path</schema_key></entry>
+    <entry><key>applets/workspace_switcher/launcher_location</key><schema_key>/schemas/apps/panel/objects/launcher_location</schema_key></entry>
+    <entry><key>applets/workspace_switcher/action_type</key><schema_key>/schemas/apps/panel/objects/action_type</schema_key></entry>
+    <entry><key>applets/workspace_switcher/text</key><schema_key>/schemas/apps/panel/objects/text</schema_key></entry>
+    <entry><key>applets/workspace_switcher/text_font</key><schema_key>/schemas/apps/panel/objects/text_font</schema_key></entry>
+    <entry><key>applets/workspace_switcher/boundary_box</key><schema_key>/schemas/apps/panel/objects/boundary_box</schema_key></entry>
 
   <!-- Notification Area Applet -->
 
@@ -449,14 +431,14 @@
       <key>applets/notification_area/toplevel_id</key>
       <schema_key>/schemas/apps/panel/objects/toplevel_id</schema_key>
       <value>
-        <string>top_panel</string>
+        <string>bottom_panel</string>
       </value>
     </entry>
     <entry>
       <key>applets/notification_area/position</key>
       <schema_key>/schemas/apps/panel/objects/position</schema_key>
       <value>
-        <int>3</int>
+        <int>2</int>
       </value>
     </entry>
     <entry>
@@ -489,168 +471,123 @@
     <entry><key>applets/notification_area/menu_path</key><schema_key>/schemas/apps/panel/objects/menu_path</schema_key></entry>
     <entry><key>applets/notification_area/launcher_location</key><schema_key>/schemas/apps/panel/objects/launcher_location</schema_key></entry>
     <entry><key>applets/notification_area/action_type</key><schema_key>/schemas/apps/panel/objects/action_type</schema_key></entry>
+    <entry><key>applets/notification_area/text</key><schema_key>/schemas/apps/panel/objects/text</schema_key></entry>
+    <entry><key>applets/notification_area/text_font</key><schema_key>/schemas/apps/panel/objects/text_font</schema_key></entry>
+    <entry><key>applets/notification_area/boundary_box</key><schema_key>/schemas/apps/panel/objects/boundary_box</schema_key></entry>
 
-  <!-- Show Desktop Applet -->
+  <!-- Mixer Applet -->
 
     <entry>
-      <key>applets/show_desktop_button/object_type</key>
+      <key>applets/mixer/object_type</key>
       <schema_key>/schemas/apps/panel/objects/object_type</schema_key>
       <value>
         <string>bonobo-applet</string>
       </value>
     </entry>
     <entry>
-      <key>applets/show_desktop_button/toplevel_id</key>
+      <key>applets/mixer/toplevel_id</key>
       <schema_key>/schemas/apps/panel/objects/toplevel_id</schema_key>
       <value>
         <string>bottom_panel</string>
       </value>
     </entry>
     <entry>
-      <key>applets/show_desktop_button/position</key>
-      <schema_key>/schemas/apps/panel/objects/position</schema_key>
-      <value>
-        <int>0</int>
-      </value>
-    </entry>
-    <entry>
-      <key>applets/show_desktop_button/panel_right_stick</key>
-      <schema_key>/schemas/apps/panel/objects/panel_right_stick</schema_key>
-      <value>
-        <bool>false</bool>
-      </value>
-    </entry>
-    <entry>
-      <key>applets/show_desktop_button/locked</key>
-      <schema_key>/schemas/apps/panel/objects/locked</schema_key>
-      <value>
-        <bool>true</bool>
-      </value>
-    </entry>
-    <entry>
-      <key>applets/show_desktop_button/bonobo_iid</key>
-      <schema_key>/schemas/apps/panel/objects/bonobo_iid</schema_key>
-      <value>
-        <string>OAFIID:GNOME_ShowDesktopApplet</string>
-      </value>
-    </entry>
-
-    <entry><key>applets/show_desktop_button/attached_toplevel_id</key><schema_key>/schemas/apps/panel/objects/attached_toplevel_id</schema_key> </entry>
-    <entry><key>applets/show_desktop_button/tooltip</key><schema_key>/schemas/apps/panel/objects/tooltip</schema_key></entry>
-    <entry><key>applets/show_desktop_button/use_custom_icon</key><schema_key>/schemas/apps/panel/objects/use_custom_icon</schema_key></entry>
-    <entry><key>applets/show_desktop_button/custom_icon</key><schema_key>/schemas/apps/panel/objects/custom_icon</schema_key></entry>
-    <entry><key>applets/show_desktop_button/use_menu_path</key><schema_key>/schemas/apps/panel/objects/use_menu_path</schema_key></entry>
-    <entry><key>applets/show_desktop_button/menu_path</key><schema_key>/schemas/apps/panel/objects/menu_path</schema_key></entry>
-    <entry><key>applets/show_desktop_button/launcher_location</key><schema_key>/schemas/apps/panel/objects/launcher_location</schema_key></entry>
-    <entry><key>applets/show_desktop_button/action_type</key><schema_key>/schemas/apps/panel/objects/action_type</schema_key></entry>
-
-  <!-- Window List Applet -->
-
-    <entry>
-      <key>applets/window_list/object_type</key>
-      <schema_key>/schemas/apps/panel/objects/object_type</schema_key>
-      <value>
-        <string>bonobo-applet</string>
-      </value>
-    </entry>
-    <entry>
-      <key>applets/window_list/toplevel_id</key>
-      <schema_key>/schemas/apps/panel/objects/toplevel_id</schema_key>
-      <value>
-        <string>bottom_panel</string>
-      </value>
-    </entry>
-    <entry>
-      <key>applets/window_list/position</key>
+      <key>applets/mixer/position</key>
       <schema_key>/schemas/apps/panel/objects/position</schema_key>
       <value>
         <int>1</int>
       </value>
     </entry>
     <entry>
-      <key>applets/window_list/panel_right_stick</key>
+      <key>applets/mixer/panel_right_stick</key>
       <schema_key>/schemas/apps/panel/objects/panel_right_stick</schema_key>
       <value>
-        <bool>false</bool>
+        <bool>true</bool>
       </value>
     </entry>
     <entry>
-      <key>applets/window_list/locked</key>
+      <key>applets/mixer/locked</key>
       <schema_key>/schemas/apps/panel/objects/locked</schema_key>
       <value>
         <bool>true</bool>
       </value>
     </entry>
     <entry>
-      <key>applets/window_list/bonobo_iid</key>
+      <key>applets/mixer/bonobo_iid</key>
       <schema_key>/schemas/apps/panel/objects/bonobo_iid</schema_key>
       <value>
-        <string>OAFIID:GNOME_WindowListApplet</string>
+        <string>OAFIID:GNOME_MixerApplet</string>
       </value>
     </entry>
 
-    <entry><key>applets/window_list/attached_toplevel_id</key><schema_key>/schemas/apps/panel/objects/attached_toplevel_id</schema_key> </entry>
-    <entry><key>applets/window_list/tooltip</key><schema_key>/schemas/apps/panel/objects/tooltip</schema_key></entry>
-    <entry><key>applets/window_list/use_custom_icon</key><schema_key>/schemas/apps/panel/objects/use_custom_icon</schema_key></entry>
-    <entry><key>applets/window_list/custom_icon</key><schema_key>/schemas/apps/panel/objects/custom_icon</schema_key></entry>
-    <entry><key>applets/window_list/use_menu_path</key><schema_key>/schemas/apps/panel/objects/use_menu_path</schema_key></entry>
-    <entry><key>applets/window_list/menu_path</key><schema_key>/schemas/apps/panel/objects/menu_path</schema_key></entry>
-    <entry><key>applets/window_list/launcher_location</key><schema_key>/schemas/apps/panel/objects/launcher_location</schema_key></entry>
-    <entry><key>applets/window_list/action_type</key><schema_key>/schemas/apps/panel/objects/action_type</schema_key></entry>
+    <entry><key>applets/mixer/attached_toplevel_id</key><schema_key>/schemas/apps/panel/objects/attached_toplevel_id</schema_key> </entry>
+    <entry><key>applets/mixer/tooltip</key><schema_key>/schemas/apps/panel/objects/tooltip</schema_key></entry>
+    <entry><key>applets/mixer/use_custom_icon</key><schema_key>/schemas/apps/panel/objects/use_custom_icon</schema_key></entry>
+    <entry><key>applets/mixer/custom_icon</key><schema_key>/schemas/apps/panel/objects/custom_icon</schema_key></entry>
+    <entry><key>applets/mixer/use_menu_path</key><schema_key>/schemas/apps/panel/objects/use_menu_path</schema_key></entry>
+    <entry><key>applets/mixer/menu_path</key><schema_key>/schemas/apps/panel/objects/menu_path</schema_key></entry>
+    <entry><key>applets/mixer/launcher_location</key><schema_key>/schemas/apps/panel/objects/launcher_location</schema_key></entry>
+    <entry><key>applets/mixer/action_type</key><schema_key>/schemas/apps/panel/objects/action_type</schema_key></entry>
+    <entry><key>applets/mixer/text</key><schema_key>/schemas/apps/panel/objects/text</schema_key></entry>
+    <entry><key>applets/mixer/text_font</key><schema_key>/schemas/apps/panel/objects/text_font</schema_key></entry>
+    <entry><key>applets/mixer/boundary_box</key><schema_key>/schemas/apps/panel/objects/boundary_box</schema_key></entry>
 
-  <!-- Workspace Switcher Applet -->
+  <!-- Trash Applet -->
 
     <entry>
-      <key>applets/workspace_switcher/object_type</key>
+      <key>applets/trash/object_type</key>
       <schema_key>/schemas/apps/panel/objects/object_type</schema_key>
       <value>
         <string>bonobo-applet</string>
       </value>
     </entry>
     <entry>
-      <key>applets/workspace_switcher/toplevel_id</key>
+      <key>applets/trash/toplevel_id</key>
       <schema_key>/schemas/apps/panel/objects/toplevel_id</schema_key>
       <value>
         <string>bottom_panel</string>
       </value>
     </entry>
     <entry>
-      <key>applets/workspace_switcher/position</key>
+      <key>applets/trash/position</key>
       <schema_key>/schemas/apps/panel/objects/position</schema_key>
       <value>
         <int>0</int>
       </value>
     </entry>
     <entry>
-      <key>applets/workspace_switcher/panel_right_stick</key>
+      <key>applets/trash/panel_right_stick</key>
       <schema_key>/schemas/apps/panel/objects/panel_right_stick</schema_key>
       <value>
         <bool>true</bool>
       </value>
     </entry>
     <entry>
-      <key>applets/workspace_switcher/locked</key>
+      <key>applets/trash/locked</key>
       <schema_key>/schemas/apps/panel/objects/locked</schema_key>
       <value>
         <bool>true</bool>
       </value>
     </entry>
     <entry>
-      <key>applets/workspace_switcher/bonobo_iid</key>
+      <key>applets/trash/bonobo_iid</key>
       <schema_key>/schemas/apps/panel/objects/bonobo_iid</schema_key>
       <value>
-        <string>OAFIID:GNOME_WorkspaceSwitcherApplet</string>
+        <string>OAFIID:GNOME_Panel_TrashApplet</string>
       </value>
     </entry>
 
-    <entry><key>applets/workspace_switcher/attached_toplevel_id</key><schema_key>/schemas/apps/panel/objects/attached_toplevel_id</schema_key> </entry>
-    <entry><key>applets/workspace_switcher/tooltip</key><schema_key>/schemas/apps/panel/objects/tooltip</schema_key></entry>
-    <entry><key>applets/workspace_switcher/use_custom_icon</key><schema_key>/schemas/apps/panel/objects/use_custom_icon</schema_key></entry>
-    <entry><key>applets/workspace_switcher/custom_icon</key><schema_key>/schemas/apps/panel/objects/custom_icon</schema_key></entry>
-    <entry><key>applets/workspace_switcher/use_menu_path</key><schema_key>/schemas/apps/panel/objects/use_menu_path</schema_key></entry>
-    <entry><key>applets/workspace_switcher/menu_path</key><schema_key>/schemas/apps/panel/objects/menu_path</schema_key></entry>
-    <entry><key>applets/workspace_switcher/launcher_location</key><schema_key>/schemas/apps/panel/objects/launcher_location</schema_key></entry>
-    <entry><key>applets/workspace_switcher/action_type</key><schema_key>/schemas/apps/panel/objects/action_type</schema_key></entry>
+    <entry><key>applets/trash/attached_toplevel_id</key><schema_key>/schemas/apps/panel/objects/attached_toplevel_id</schema_key> </entry>
+    <entry><key>applets/trash/tooltip</key><schema_key>/schemas/apps/panel/objects/tooltip</schema_key></entry>
+    <entry><key>applets/trash/use_custom_icon</key><schema_key>/schemas/apps/panel/objects/use_custom_icon</schema_key></entry>
+    <entry><key>applets/trash/custom_icon</key><schema_key>/schemas/apps/panel/objects/custom_icon</schema_key></entry>
+    <entry><key>applets/trash/use_menu_path</key><schema_key>/schemas/apps/panel/objects/use_menu_path</schema_key></entry>
+    <entry><key>applets/trash/menu_path</key><schema_key>/schemas/apps/panel/objects/menu_path</schema_key></entry>
+    <entry><key>applets/trash/launcher_location</key><schema_key>/schemas/apps/panel/objects/launcher_location</schema_key></entry>
+    <entry><key>applets/trash/action_type</key><schema_key>/schemas/apps/panel/objects/action_type</schema_key></entry>
+    <entry><key>applets/trash/text</key><schema_key>/schemas/apps/panel/objects/text</schema_key></entry>
+    <entry><key>applets/trash/text_font</key><schema_key>/schemas/apps/panel/objects/text_font</schema_key></entry>
+    <entry><key>applets/trash/boundary_box</key><schema_key>/schemas/apps/panel/objects/boundary_box</schema_key></entry>
 
   </entrylist>
 
